test_runner/fixtures/neon_fixtures.py

```diff
@@ -2042,15 +2042,40 @@ def __init__(
         proxy_port: int,
         http_port: int,
         mgmt_port: int,
+        external_http_port: int,
         auth_backend: NeonProxy.AuthBackend,
         metric_collection_endpoint: Optional[str] = None,
         metric_collection_interval: Optional[str] = None,
     ):
         host = "127.0.0.1"
-        super().__init__(dsn=auth_backend.default_conn_url, host=host, port=proxy_port)
+        domain = "proxy.localtest.me" # resolves to 127.0.0.1
+        super().__init__(dsn=auth_backend.default_conn_url, host=domain, port=proxy_port)
 
+        subprocess.run(
```

Comment:
check the output code

and running processing in `__init__` is not a common practice.

Reply:
moved to start and checked ret code